contracts/voting/cwd-voting-cw20-staked/src/contract.rs

```diff
@@ -407,12 +522,52 @@ pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractEr
 
                     // Save staking contract addr
                     STAKING_CONTRACT.save(deps.storage, &staking_contract_addr)?;
+                    let vesting_info = VESTING_INFO.load(deps.storage)?;
```

Comment:
what is the point of keeping the vesting info in state? can't imagine we need this after instantiation?

Reply:
We either instantiate the vest contract immediately or we intantiate it in the reply after instantiating the cw20 and/or the cw20-stake. If we're instantiating in the reply we need to some how persist or transfer vest_info to that reply section. I saw this pattern being used in cw20-stake so I copied it.